src/main/java/gov/usds/case_issues/authorization/NamedOAuth2User.java

```diff
@@ -54,6 +47,22 @@ public String getName() {
 	}
 
 	public String toString() {
-		return getName();
+		return name;
+	}
+
+	private void updateUsers(UserRepository userRepo) {
+		User existingUser = userRepo.findByUserId(name);
+		if (existingUser != null) {
+			existingUser.setlastActive();
+			userRepo.save(existingUser);
+			return;
+		}
+		User newUser = new User(name, getAttributeName());
+		userRepo.save(newUser);
+	}
+
+	private String getAttributeName() {
+		Object attributeName = attributes.get("name");
```

Comment:
I think this appeared in a couple of different places in the initial-implementation OAuth object, which suggests that we might want it to be at a configurable location. I'm not sure we need to deal with that right now, though.

Reply:
added a ticket for this https://trello.com/c/Guqd4QdV